src/components/dropdown/Dropdown.tsx

```diff
@@ -78,6 +78,8 @@ export type DropdownProps<T> = {
   inputValue?: string;
   /** Maximum height of the menu before scrolling */
   maxMenuHeight?: number;
+  /** Max height of the options input before scrolling */
+  maxSelectHeight?: number;
```

Comment:
why not maxHeight directly? 
@AliciaSymphony @symphony-jean-michael  wdyt

Reply:
Sounds good